lib/mutant/mutator/node/named_value/access.rb

```diff
@@ -19,6 +19,44 @@ def dispatch
             emit_singletons
           end
 
+          # Named value access emitter for instance variables
+          class Ivar < Access
+
+            handle(:ivar)
+
+            children :name
+
+            # Emit mutations
+            #
+            # @return [undefined]
+            #
+            # @api private
+            def dispatch
+              emit_attribute
+              super()
+            end
+
+            private
+
+            # Emit instance variable as attribute send
+            #
+            # @return [undefined]
+            #
+            # @api private
+            def emit_attribute
+              emit(s(:send, nil, attribute_name))
+            end
+
+            # Variable name without leading '@'
+            #
+            # @return [Symbol]
+            #
+            # @api private
+            def attribute_name
+              name.slice(1..-1).to_sym
```

Comment:
I think the range literal should be moved to a constant.

We typically use constants for all mutable literals, unless the propose of the literal is to create a mutable value.